vllm/executor/executor_base.py

```diff
@@ -6,7 +6,7 @@
 from abc import ABC, abstractmethod
```

Comment:
ignore file, to be rebased once https://github.com/vllm-project/vllm/pull/26734 lands